Sources_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -126,6 +126,12 @@ extension ChannelDTO {
         request.predicate = NSPredicate(format: "ANY queries.filterHash == %@", query.filter.filterHash)
         return request
     }
+
+    static var channelListFetchRequestAll: NSFetchRequest<ChannelDTO> {
+        let request = NSFetchRequest<ChannelDTO>(entityName: ChannelDTO.entityName)
+        request.sortDescriptors = [ChannelListSortingKey.defaultSortDescriptor]
+        return request
+    }
```

Comment:
Or this can be channels without any filterHash

Reply:
Nice!